docs/UserGuide.adoc

```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
+ORGANice will prevent you from deleting persons in the following categories:
+
+* They are a patient and their status is currently under processing.
+* They are a donor and their status is currently under processing.
+* They are a doctor and they are in charge of some number of patients in ORGANice.
+
+If you want to delete a patient that has NRIC of S4612296H and is not currently under processing, type the following command: `delete S4612296H`
+
+// end::delete[]
+=== Clearing all entries : `clear`
+
+You can clear all entries in ORGANice by typing `clear`. You should run clear to start a fresh version of ORGANice after you have familiarised yourself with the commands using our dummy data.
```

Comment:
You can clear all data in ORGANice by typing `clear`. You could use this command to remove all dummy data after you have familiarized yourself with how ORGANice works.